conf/nginx.conf.server

```diff
@@ -13,8 +13,8 @@ proxy_buffering on;
 
 ## Configures expiration for upstream content based on content type.
 map $upstream_http_content_type $content_type_expiry {
-    default                                 10s;
-    'application/json'                      10s;
+    default                                 5s;
```

Comment:
This time was arbitrary since we want to expire basically immediately. Lowered to 5 to shave some time off test runs.